gui/middleware/form.py

```diff
@@ -48,4 +59,11 @@ def __update(self, *args, **kwargs):
             args = (self.instance.id, update) + args
 
         with client as c:
-            return c.call(f"{self.middleware_plugin}.update", *args, **kwargs)
+            try:
+                return c.call(f"{self.middleware_plugin}.update", *args, **kwargs)
+            except ClientException as e:
```

Comment:
Using regex for this makes me twitch. Is using `errno` a bad idea here?

Reply:
Which `errno` do you propose here?

Reply:
I have no preference, it could be a custom `errno`, or `EPROCUNAVAIL`

Reply:
@william-gr I doubt we should use codes like `EPROCUNAVAIL` for our own purposes, it may confuse `midclt` users. Custom `errno` would be a string, like `ESERVICESTARTFAILURE`, right?

Reply:
Its a [number, string] pair. Its currently defined in service_exception.py

Reply:
Fixed in 9342cea213e37b768599aa57b30caf3759d24a09